sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3954,6 +3954,17 @@ object functions {
   def to_json(e: Column): Column =
     to_json(e, Map.empty[String, String])
 
+  /**
```

Comment:
Let's don't add it here per https://github.com/apache/spark/blob/d8ec9504e27a4097d7b0997d5601105607f26dec/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L42-L60

Reply:
I believe it should be allowed to be used directly.